MAUI/ComboBox/UI-Customization.md

```diff
@@ -698,3 +698,29 @@ Completed event can be subscribed in C# also:
 The following image illustrates the result of the above code:
 
 ![.NET MAUI ComboBox completed event](Images/UICustomization/CompletedEvent.png)
+
+## CursorPosition
+
+The cursor position in the input view can either be obtained or updated using the `CursorPosition` property in SfComboBox.
```

Comment:
The cursor position in the input view can be obtained or updated using the `CursorPosition` property in the SfComboBox.